src/services/grantee.js

```diff
@@ -59,83 +59,72 @@ export async function granteeById(granteeId, grantScopes) {
 export async function granteesByName(query, scopes, sortBy, direction, offset) {
   // fix the query
   const q = `%${query}%`;
+  const limit = GRANTEES_PER_PAGE;
 
-  // first get all grants with numbers that match the query string
-  const matchingGrantNumbers = await Grant.findAll({
-    attributes: [],
-    where: {
-      number: {
-        [Op.iLike]: q, // sequelize automatically escapes this
-      },
-      status: 'Active',
-      [Op.and]: scopes,
-    },
-    include: [
-      {
-        model: Grantee,
-        as: 'grantee',
-        attributes: ['id'],
-      },
+  const rows = await Grantee.findAll({
+    attributes: [
+      [sequelize.literal('DISTINCT COUNT(*) OVER()'), 'count'],
+      [sequelize.fn('STRING_AGG', sequelize.fn('DISTINCT', sequelize.col('grants.programSpecialistName')), ', '), 'programSpecialists'],
```

Comment:
will these be sorted as they were previously?

Reply:
Good catch, they are now explicitly ordered.